docs/source/key_components/dashboard_visualization.rst

```diff
@@ -0,0 +1,109 @@
+Dashboard Visualization
+=======================
+
+Env-dashboard is a post-experiment visualization tool, aims to provide
```

Comment:
Is ``Env-dashboard`` the name of the visualization tool?

Reply:
visualization tool has multiple parts, env refers to the mode, dashboard refers to the way to display data.